distribution/docker/solr/README.md

```diff
@@ -0,0 +1,32 @@
+# DDF Compatible Solr Container
```

Comment:
Can you add a section describing the purpose and use of the Maven `docker` profile?

Reply:
I'm not sure I want to even mention the profile since eventually it will go away. Right now its just a transparent thing that is auto-activated on the correct platforms

Reply:
Can we mention that the profile  will go away because of _____.

Reply:
Yep, I actually did add a comment about it inside both pom files